src/main/java/org/myrobotlab/service/LocalSpeech.java

```diff
@@ -30,7 +31,7 @@
   private static final long serialVersionUID = 1L;
 
   public final static Logger log = LoggerFactory.getLogger(LocalSpeech.class);
-  
+  private String ttsPath = "\"" + System.getProperty("user.dir") + File.separator + "tts" + File.separator + "tts.exe" + "\"";

```

Comment:
add a setter/getter for ttsPath so people can override it if they install tts.exe into a different location.